### PR TITLE
Implement twitter authorization

### PR DESCRIPTION
**Pull request by jacqu3lin3**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,14 +1,42 @@
 require("express-async-errors");
 const express = require("express");
 const path = require("path");
+const passport = require('passport');
```
Comment (usmansbk) [inline server.js:4]: Please add a link to the Twitter clubhouse ticket

Comment (jacqu3lin3) [reply server.js:4]: Here it is: https://app.clubhouse.io/startng/story/43811/register-route-implement-twitter-auth

Comment (jacqu3lin3) [reply server.js:4]: Sorry, I'm a bit confused. Did you mean I should add the link within the code? If so, where would the link go in there?

File: server.js [hunk 2]
```diff
@@ -1,14 +1,42 @@
...
 const app = express();
 
+const port = 2020 || process.env.PORT;
 const preMiddlewares = require("./src/middlewares/preMiddlewares");
 const errorMiddlewares = require("./src/middlewares/errorMiddlewares");
 const routes = require("./src/routes");
 
 preMiddlewares(app);
 
+passport.use(new Strategy({
```
Comment (usmansbk) [inline server.js]: can you move this logic to preMiddlewares

Comment (jacqu3lin3) [reply server.js]: Done.

Comment (jacqu3lin3) [reply server.js]: @usmansbk

**Pull request merged**